testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/MsalWrapper.java

```diff
@@ -421,10 +421,10 @@ public void onAccountLoaded(IAccount activeAccount) {
 
                     @Override
                     public void onAccountChanged(IAccount priorAccount, IAccount currentAccount) {
-                        notifyCallback.notify("single account is changed from " +
-                                (priorAccount == null ? "null" : priorAccount.getUsername()) +
-                                " to " +
-                                (currentAccount == null ? "null" : currentAccount.getUsername()));
+                        notifyCallback.notify("single account is changed from "
```

Comment:
TODO How does this work with guest accounts?

Reply:
If the root profile doesn't exist, we could try iterating through the profiles to find the username.

Reply:
Resolved in c30ce285f18799c6253e8748f6da83df36c8f230